variables.tf

```diff
@@ -489,6 +484,7 @@ variable "repositories" {
       description = "Mage-OS Mirror & Nightly Build Instances In One Click"
       teams       = ["infrastructure"]
       topics      = ["mage-os", "magento", "ecommerce", "magento2", "adobecommerce"]
+      archived     = true
```

Comment:
```suggestion
      archived    = true
```

Reply:
@rhoerr, somehow, I cannot commit my suggestion here at the moment. Please fix :pray:

Reply:
done it

Reply:
nvm 
![CleanShot 2025-01-14 at 09 46 46](https://github.com/user-attachments/assets/2d73b4dd-c8cd-4299-a64e-0228267819d7)

Reply:
@rhoerr if you commit this suggestion we can merge and close right away :-)

Reply:
I can't either. 😄 It must be a protected branch now. I'll have to fix it as `mage-os-ci` later. My mistake, I could've sworn the indenting was right.